packages/web-components/@groupby/elements-products/test/unit/products.test.ts

```diff
@@ -67,26 +97,66 @@ describe('Products Sayt Component', () => {
   itShouldExtendClass(() => component, ProductsBase);
 
   describe('connectedCallback', () => {
+    let cacheResponseEventName;
+
+    beforeEach(() => {
+      cacheResponseEventName = component.cacheResponseEventName = 'sayt-cache-response';
```

Comment:
If you take the getter approach, you can use the getter directly without having to stub it.

For the record, if you were to keep this, it would be better style to make `cacheResponseEventName` a `const` and simply set `component.cacheResponseEventName = cacheResponseEventName` here.